<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>File</class>
 <widget class="QWidget" name="File">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-top-left-radius:10px;
border-top-right-radius:10px;
border-bottom-left-radius: 10px;
border-bottom-right-radius: 10px;</string>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>481</y>
     <width>941</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonDownLoad">
   <property name="geometry">
    <rect>
     <x>873</x>
     <y>262</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>下载</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonUpLoad">
   <property name="geometry">
    <rect>
     <x>873</x>
     <y>126</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>上传</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonDelete">
   <property name="geometry">
    <rect>
     <x>873</x>
     <y>386</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelLocalPath">
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>240</y>
     <width>751</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>this is path</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="TableTarget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>73</y>
     <width>826</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 243, 233);</string>
   </property>
   <column>
    <property name="text">
     <string>文件名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>类型</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>大小</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="LabelTargetPath">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>53</y>
     <width>821</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>this is path</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="TableLocal">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>831</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 243, 233);</string>
   </property>
   <column>
    <property name="text">
     <string>文件名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>类型</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>大小</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="ButtonExit_2">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>0</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Minimize_Window_128px_1178220_easyicon.net.ico</normaloff>:/Minimize_Window_128px_1178220_easyicon.net.ico</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonExit">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>0</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/exit_quit_24px_16971_easyicon.net.ico</normaloff>:/exit_quit_24px_16971_easyicon.net.ico</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>985</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>progressBar</zorder>
  <zorder>ButtonDownLoad</zorder>
  <zorder>ButtonUpLoad</zorder>
  <zorder>ButtonDelete</zorder>
  <zorder>LabelLocalPath</zorder>
  <zorder>TableTarget</zorder>
  <zorder>LabelTargetPath</zorder>
  <zorder>TableLocal</zorder>
  <zorder>label_2</zorder>
  <zorder>ButtonExit</zorder>
  <zorder>ButtonExit_2</zorder>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
